cloud-sql-proxy/cloud-sql-proxy.sh

```diff
@@ -259,12 +323,18 @@ function main() {
       else
         echo "Service hive-metastore is not enabled"
       fi
-      systemctl stop mysql
-      systemctl disable mysql
+      if ( systemctl is-enabled --quiet mysql ); then
+        systemctl stop mysql
+        systemctl disable mysql
+      fi
     fi
     install_cloud_sql_proxy
     if [[ $enable_cloud_sql_metastore = "true" ]]; then
-      configure_sql_client
+      if [[ "${MASTER_INDEX?}" == 0 ]]; then
```

Comment:
You can get rid of several variables by making this `if [[ "${HOSTNAME}" == "${DATAPROC_MASTER}" ]]`. We do that in a couple other places, such as https://github.com/GoogleCloudPlatform/dataproc-initialization-actions/blob/master/jupyter2/jupyter2.sh#L109.

Reply:
thanks for the tips, I've applied the changes.